Cargo.toml

```diff
@@ -129,7 +129,7 @@ proc-macro-error = { version = "1.0", default-features = false }
 proc-macro2 = "1.0"
 procfs = "0.16.0"
 pwd = "1.3"
-quick-xml = "0.32.0"
+quick-xml = ">= 0.33.0, < 0.38.0"
```

Comment:
why not just 0.38.0?

Reply:
0.38.0 is not out yet, and since nu-command builds fine with 0.37.0 I'm assuming it will work with future 0.37.x versions too

https://crates.io/crates/quick-xml/

Reply:
Sure, I can certainly update to 0.37.0 only. I wasn't expecting the wide range to land as is, I just wanted to let the upstream devs pick which version to go with.

I can get Fedora's quick-xml updated to 0.37.0 later (or whichever version nushell is using the next time we rebase our package)

Reply:
@fdncred updated the commit, commit message, and PR summary and description to use 0.37.0.